system-linux.c

```diff
@@ -3805,10 +3805,12 @@ static int system_add_ip6_tunnel(const char *name, const unsigned int link,
 			nla_nest_end(nlm, fmrs);
 		}
 #endif
-		if (tun_flags)
-			nla_put_u32(nlm, IFLA_IPTUN_FLAGS, tun_flags);
 	}
 
+	tun_flags |= IP6_TNL_F_USE_ORIG_TCLASS;
```

Comment:
This bitwise-OR makes the subsequent `if (tun_flags)` condition useless, can you remove the condition so that the code reads

```c
tun_flags |= IP6_TNL_F_USE_ORIG_TCLASS;
nla_put_u32(nlm, IFLA_IPTUN_FLAGS, tun_flags);
```

or even more terse:

```
nla_put_u32(nlm, IFLA_IPTUN_FLAGS, tun_flags | IP6_TNL_F_USE_ORIG_TCLASS);
```

Reply:
thanks for the prompt review @jow- , comment addressed and code updated.